src/js/Controllers/UIController.js

```diff
@@ -203,6 +203,18 @@ class UIController {
                 }
                 this.listener.send(RpcFactory.SuccessResponse(rpc));
                 return null
+            case "UnsubscribeButton":
+                    store.dispatch(subscribeButton(
+                        rpc.params.appID,
+                        rpc.params.buttonName,
+                        false
+                    ));
+                    if(!rpc.params.buttonName || !rpc.params.appID) {
+                        this.listener.send(RpcFactory.ErrorResponse(rpc, GENERIC_ERROR, "No button provide to unsubscribe"));
+                        return;
+                    };
+                    this.listener.send(RpcFactory.SuccessResponse(rpc));
+                    return null
```

Comment:
@LiliiaShlikhtLuxoft Is there a reason to return `null` and return `void` two lines of code above?

Reply:
@ValeriiMalkov in UIController in handleRPC function have a lot of switch cases. I used return null just to break this case.

Reply:
@LiliiaShlikhtLuxoft My question is: Why in line number `214` returned void and in line number `217` returned null ?

Reply:
@ValeriiMalkov in 214 line I need just return, in 217 it was copy-pasted from similar case. Do you think return null should be changed to something else?

Reply:
@LiliiaShlikhtLuxoft Why do you need just return, but in other case return `null` ? Are these different cases ?

Reply:
Fixed